d2l-dom.js

```diff
@@ -91,6 +91,8 @@ D2L.Dom = {
 		while (currentNode) {
 			if (currentNode instanceof ShadowRoot) {
 				currentNode = this.getComposedParent(currentNode);
+			} else if (currentNode instanceof DocumentFragment) {
+				return null;
```

Comment:
Looks good.  Note: it looks like `tagName === 'BODY'` can be moved up as well, which would eliminate a `getComputedStyle` call (which can be expensive) if/when the `body` element is reached.
```
if (currentNode instanceof ShadowRoot) {
	currentNode = this.getComposedParent(currentNode);
} else if (currentNode instanceof DocumentFragment) {
	return null;
} else if (currentNode.tagName === 'BODY') {
	return currentNode; 
}
```